rest-service/manager_rest/server.py

```diff
@@ -88,6 +88,31 @@ def cope_with_db_failover():
                 attempt, max_attempts, err,
             )
 
+def query_service_settings():
+    """Check for when was the config updated, and if needed, reload it.
+
+    This makes sure that config updates will (eventually) be propagated
+    to all workers, and that every worker always has the most recent
+    config/permissions settings available.
+    """
+    last_updated_subquery = (
+        db.session.query(models.Config.updated_at.label('updated_at'))
+        .union_all(
+            db.session.query(models.Role.updated_at.label('updated_at'))
+        ).subquery()
+    )
+    config_last_updated = db.session.query(
+        db.func.max(last_updated_subquery.c.updated_at)
+    ).scalar()
+    current_app.logger.debug('Last updated locally: %s, in db: %s',
+                             config.instance.last_updated,
+                            config_last_updated)
+    if config_last_updated is not None and (
```

Comment:
Not a big issue, but this might be clearer if config_last_updated was called db_config_last_updated- on first glance it looked like this line and the next line were referring to the same variable.